node/core/provisioner/src/lib.rs

```diff
@@ -269,6 +269,13 @@ impl ProvisionerJob {
 			self.metrics.on_inherent_data_request(Err(()));
 		} else {
 			self.metrics.on_inherent_data_request(Ok(()));
+			gum::debug!(
+				target: LOG_TARGET,
+				len_signed_bitfield = self.signed_bitfields.len(),
+				len_backed_candidates = self.backed_candidates.len(),
```

Comment:
Since this is only executed every 6s by one validator at `debug!` level, I think we can afford printing some additional data.